README.md

```diff
@@ -1,118 +1,81 @@
-Container and runtime basic services
-* **nuxeo-core**:
-Document/content management core services
-* **nuxeo-services**:
-Basic services such as file manager, directories, document types
-* **nuxeo-theme**:
-Services related to the theme and theme rendering
-* **nuxeo-jsf**:
-JSF related services
-* **nuxeo-webengine**:
-Services and framework related to WebEngine, the Nuxeo lightweight rendering engine
-* **nuxeo-features**:
-Advanced high-level services, such as audit, imaging, publisher, thumbnails, search
-* **nuxeo-dm**:
-The default Nuxeo Platform application, mostly configuration and UI elements
-* **nuxeo-distribution**:
-This module builds, packages and tests the Nuxeo products.
-
-# Building
-
-Open Nuxeo in [Eclipse Che](https://www.eclipse.org/che/):  
-[![Open in Eclipse Che](http://beta.codenvy.com/factory/resources/codenvy-contribute.svg)](https://beta.codenvy.com/f?id=r3pbh9wv7tjvbws7)
-
-## Requirements
-
-Running the Nuxeo Platform requires Java 11.
-Depending on the features you want to use, you may need some third-party software, such as Libre Office and pdftohtml for document preview or ImageMagick for pictures. The list of third-party software is available in our Admin documentation: [Installing and Setting Up Related Software](http://doc.nuxeo.com/x/zgJc).
+## Installation
 
-Building the Nuxeo Platform requires the following tools:
+Please follow the [Installation](https://doc.nuxeo.com/n/b1j) guide.
 
-* JDK 11 (Oracle's JDK or OpenJDK recommended)
-* Apache Maven 3.1.1+ (3.2+ recommended)
-* Apache Ant 1.7.1+
-* Git (obviously)
+Running the Nuxeo Platform requires JDK 11, Oracle's JDK or OpenJDK.
 
-# QA
+Depending on the features you want to use, you may need some third-party software, such as LibreOffice and pdftohtml for document preview or ImageMagick for pictures. The list of third-party software is available in our Admin documentation: [Installing and Setting Up Related Software](https://doc.nuxeo.com/n/Yki).
 
-Each module includes unit and integration tests. Functional tests are available in nuxeo-distribution and for each addon package module.
+## Building
 
-We also provide some tooling for tests:
+Building the Nuxeo Platform requires the following tools:
 
-* [https://github.com/nuxeo/tools-nuxeo-ftest](https://github.com/nuxeo/tools-nuxeo-ftest)
-* [https://github.com/nuxeo/ant-assembly-maven-plugin](https://github.com/nuxeo/ant-assembly-maven-plugin)
-* [https://github.com/nuxeo/integration-scripts/](https://github.com/nuxeo/integration-scripts/)
+- Git (obviously)
+- JDK 11 (Oracle's JDK or OpenJDK recommended)
+- Apache Maven 3.1.1+ (3.2+ recommended)
+- Apache Ant 1.7.1+
 
-# Deploying
+Get the source code:
 
-1. Get the source code:
-```
+```shell
 git clone git@github.com:nuxeo/nuxeo.git
 cd nuxeo
-python clone.py master -a
 ```
-2. Build using Maven:
+
+To build everything, including the server ZIP, packages and Docker image, run:
+
+```shell
+mvn clean install -Pdistrib,docker -DskipTests
```

Comment:
The command you're suggesting doesn't build the packages, which is the purpose of this first command: build everything.

Yet, you're absolutely right about the following things to speed up the build!
- export the relevant `MAVEN_OPTS`
- no `mvn clean`
- `-Dnuxeo.skip.enforcer=true`
- `-T6`

Note that `-nsu` isn't needed since the command builds the whole repository from sources and there is no SNAPSHOT dependency.